nixos/modules/services/web-apps/lotide.nix

```diff
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lotide;
+in
+{
+  options.services.lotide = {
+    enable = mkEnableOption "the lotide service";
+
+    package = mkOption {
```

Comment:
There is only one version of `lotide` available, and there are no variants on it. This option provides no value, in my opinion.